DirectProgramming/DPC++FPGA/Tutorials/Features/dsp_control/README.md

```diff
@@ -0,0 +1,192 @@
+# DSP Control
+This FPGA tutorial demonstrates how to set implementation preference for math operations, which affects hardware resource utilization on FPGA device.
```

Comment:
This FPGA tutorial demonstrates how to set the implementation preference for certain math operations (add, subtract, and multiplication by a constant) between hardened DSP blocks and soft logic.